### DISCUSSION TITLE
"If you record a clip now..." and telling me to upgrade to Reaper 7.23

### DISCUSSION THREAD

**Discussion by jimmydeer** [helgoboss/helgobox#1184] [General]
Just installed 2.16.4-getting the "if you record" message at the top-what is this and why did it happen? 

Also "detected one issue!" that I'm only running 7.22-this is the current version, I don't really want to beta test reaper as well

**Comment by helgoboss** [comment]
This only happens if the metronome is switched off. Switch it on and you won't see it anymore. BTW, you can hold the mouse cursor over the text and have a look at the help window. That will explain things in more detail..

You will need to put up with that "Detected 1 issue" note until REAPER 7.23 is released. At the moment, this just means that the SPACE key won't do anything special if you have mapped it on REAPER as a global shortcut (which I know is the case for you). Except that, you should be fine with 7.22.

REAPER and Playtime development goes hand in hand, so it will sometimes be necessary to update both to benefit from improvements.

**Comment by jimmydeer** [comment]
Ok, thanks.
